docs/en/latest/FAQ.md

```diff
@@ -552,3 +552,20 @@ apisix:
 **Note:**
 
 - Whenever trying to connect TLS services with cosocket, you should set `apisix.ssl.ssl_trusted_certificate`
```

Comment:
Better to clarify that we only need to set trusted certificate when the certificate is untrusted.